R/pkg/R/mllib_clustering.R

```diff
@@ -404,11 +411,14 @@ setMethod("summary", signature(object = "LDAModel"),
             vocabSize <- callJMethod(jobj, "vocabSize")
             topics <- dataFrame(callJMethod(jobj, "topics", maxTermsPerTopic))
             vocabulary <- callJMethod(jobj, "vocabulary")
+            trainingLogLikelihood <- callJMethod(jobj, "trainingLogLikelihood")
+            logPrior <- callJMethod(jobj, "logPrior")
```

Comment:
My take is the convention in R for missing values in a structure (list, data.frame) is `NA`.
And since we are returning a list we could also simply omit it if the value is not applicable or available. (ie. don't add `trainingLogLikelihood` to the list in L421)

In any case, what does it look like when printing the `summary` returned here for these values (NULL, NA) we are proposing?